reference/exec/functions/proc-open.xml

```diff
@@ -49,8 +49,30 @@
      <term><parameter>cmd</parameter></term>
      <listitem>
       <para>
-       要执行的命令
+       以 &string; 形式执行的命令行。特殊字符必须经过转义，并且使用正确的引号。
       </para>
+      <note>
+       <simpara>
+        在 <emphasis>Windows</emphasis> 上, 除非在 <parameter>other_options</parameter> 中
+        把 <literal>bypass_shell</literal> 设置为  &true;
+        ，否则 <parameter>cmd</parameter> 会被传递给 <command>cmd.exe</command> （实际上是 <literal>%ComSpec%</literal>）
+        其中的 <literal>/c</literal> 标志是未加引号的字符串 （也就是和 <function>proc_open</function> 一样）。
+        这可能会导致 <command>cmd.exe</command> 删除 <parameter>cmd</parameter> 中的引号 （详见 <command>cmd.exe</command> 文档），
+        从而导致意外的，甚至是潜在的危险行为，因为
+        <command>cmd.exe</command> 错误消息可能包含 （部分） 传递的 <parameter>cmd</parameter> （见下面的例子）。
+       </simpara>
```

Comment:
```
On <emphasis>Windows</emphasis>, unless <literal>bypass_shell</literal> is set to &true; in
        <parameter>other_options</parameter>, the <parameter>cmd</parameter> is
        passed to <command>cmd.exe</command> (actually, <literal>%ComSpec%</literal>)
        with the <literal>/c</literal> flag as <emphasis>unquoted</emphasis> string
        (i.e. exactly as has been given to <function>proc_open</function>).
        This can cause <command>cmd.exe</command> to remove enclosing quotes from
        <parameter>cmd</parameter> (for details see the <command>cmd.exe</command> documentation),
        resulting in unexpected, and potentially even dangerous behavior, because
        <command>cmd.exe</command> error messages may contain (parts of) the passed
        <parameter>cmd</parameter> (see example below).
```
特殊标签的数量需要匹配，我看了下英文原文中 <emphasis>unquoted</emphasis>对应到中文中并没有添加标签，其实可以写成`<emphasis>未加引号的</emphasis>字符串 `